libcxx/docs/ReleaseNotes/20.rst

```diff
@@ -59,6 +59,16 @@ Deprecations and Removals
 Upcoming Deprecations and Removals
 ----------------------------------
 
+LLVM 20
+~~~~~~~
+
+- The ``LIBCXX_ENABLE_ASSERTIONS`` CMake variable and the ``_LIBCPP_ENABLE_ASSERTIONS`` macro that were used to enable
+  the safe mode will be removed in LLVM 20.
+
+- The C++20 synchronization library will be removed entirely in language modes prior to C++20 in LLVM 20.
+
+- The ``_LIBCPP_ENABLE_REMOVED_ALLOCATOR_CONST`` macro will no longer have an effect.
```

Comment:
Those are actually already above with TODOs, except for `_LIBCPP_ENABLE_REMOVED_ALLOCATOR_CONST`.

Reply:
Fix that. I've also copy-pasted the C++03 removal to this file.